src/common/dataHooks/useStories.js

```diff
@@ -49,7 +50,7 @@ export function useStory({ id, edit = false }) {
 export function useStoryCreate() {
   const { sitename } = useParams()
   // eslint-disable-next-line no-unused-vars
-  const [searchParams, setSearchParams] = useSearchParams()
+  const [searchParams, setSearchParams] = useSearchParams() // NOSONAR
```

Comment:
The //NOSONAR was added here and on line 80 as these lines were unchanged and causing sonar errors. Additionally the eslint-disable made it seem like the declaration should not be removed. However I can remove the unused declaration if we really aren't using it any more.

Reply:
This is a bug of sorts with react-router-dom, if we want to access `setSearchParams` as the second item in the array, we have to declare `searchParams` whether we are using it or not. The NOSONAR is fine, or we can mark it as "not going to change" in SonarCloud. Thanks for checking.